0000-settlement-engines/0000-settlement-engines.md

```diff
@@ -0,0 +1,395 @@
+4. Peer settlement engine processes the message and responds with its own message.
+5. Peer connector sends the response message back across the transport to the origin connector within an ILP Fulfill or ILP Reject, depending upon the code of the response (described below). If the peer, connector was unable to process the request, it MUST respond with an ILP Reject.
+6. Origin connector sends the response message back to the origin settlement engine.
+
+### ILP Prepare
+
+- `amount`: `0`
+- `expiresAt`: _Determined by connector_
+- `executionCondition`: `e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855`
+- `destination`: `peer.settle`
+- `data`: _Request message from sender settlement engine_
+
+### ILP Fulfill
+
+- `fulfillment`: `0000000000000000000000000000000000000000000000000000000000000000`
+- `data`: _Response message from recipient settlement engine_
+
+### ILP Reject
+
+- `code`: _Determined by connector from HTTP status of forwarded request_
+- `triggeredBy`: `peer.settle`
+- `message`: _Determined by connector_
+- `data`: _Response message from recipient settlement engine, or empty if antecedent failure_
+
+## Accounts and identifiers
+
+Each account MUST be identified by a unique, [URL-safe](https://tools.ietf.org/html/rfc3986#section-2.3) string, immutable for the lifetime of the account.
+
+The settlement engine MUST be responsible for correlating an account identifier to the peer's identity on the shared ledger or settlement system, if required. For separation of concerns between clearing and settlement, the accounting system is NOT RECOMMENDED to have knowledge of the peer's identity on the shared settlement system.
+
+## Units and quantities
+
+Asset amounts may be represented using any arbitrary denomination. For example, one U.S. dollar may be represented as \$1 or 100 cents, each of which is equivalent in value. Likewise, one Bitcoin may be represented as 1 BTC or 100,000,000 satoshis.
+
+A **standard unit** is the typical unit of value for a particular asset, such as \$1 in the case of U.S. dollars, or 1 BTC in the case of Bitcoin.
+
+A **fractional unit** represents some unit smaller than the standard unit, but with greater precision. Examples of fractional monetary units include one cent (\$0.01 USD), or 1 satoshi (0.00000001 BTC).
+
+An **asset scale** is the difference in orders of magnitude between the standard unit and a corresponding fractional unit. More formally, the asset scale is a non-negative integer (0, 1, 2, …) such that one standard unit equals the value of `10^(scale)` corresponding fractional units. If the fractional unit equals the standard unit, then the asset scale is 0.
+
+For example, one cent represents an asset scale of 2 in the case of USD, whereas one satoshi represents an asset scale of 8 in the case of Bitcoin.
+
+### Selecting scales
+
+Settlement engines are RECOMMENDED to perform or fulfill requests with amounts denominated in the same scale as its settlements.
```

Comment:
Are they? Ethereum connector accounts are with 9, but the engine uses 18

Reply:
I think this is trying to say that an SE should make payments using the same scale that it was configured with. So, if an Eth SE advertises a scale of 18, then Eth settlement payment should be made at that scale.

That said, I feel like this should be a `MUST`. Is there a scenario where an SE says, "You should perceive me as using scale of 9, but I actually settle in scale of 18"? Why would the SE want to do such a thing?